src/db.c

```diff
@@ -1022,7 +1022,7 @@ void typeCommand(client *c) {
 }
 
 void shutdownCommand(client *c) {
-    int flags = 0;
+    int flags = SHUTDOWN_WAIT_REPL;
```

Comment:
Do we also want to bypass the refusal to shutdown when writing the AOF or RDB files fail? Maybe FORCE or KILL is a more descriptive flag name then? Or do you want to separate these cases with different flags?

Reply:
my first thought is that we should have different flags.
if the user is also gonna run the risk of losing his AOF file (enabled but not yet created), it's a different than giving up on replicas, (since the configured timeout might anyway not get to do what it aims)

but i think to document that difference is too complicated, so maybe just one FORCE flag to bypass both.
let's document that detail in the top comment, and then see what others think.

Reply:
While implemented, I realized the two flags are very different, so I made separate flags: NOW and FORCE. I also removed SHUTDOWN_WAIT_REPL and made that the default, which is disabled by NOW.

I don't think it'll be hard to document it.

Reply:
LGTM